test/MeshKernelNETTest/Api/MeshKernelTest.cs

```diff
@@ -3454,6 +3454,52 @@ public void Mesh2dExpungeStateThroughApi()
                 }
             }
         }
+
+        [Test]
+        public void Mesh2dGetNodeEdgeDataThroughApi()
+        {
+
+            // Setup
+            using (DisposableMesh2D mesh = CreateMesh2D(10, 10, 10, 10))
+            using (var api = new MeshKernelApi())
+            {
+                var id = 0;
+                var meshRetrived = new DisposableMesh2D();
+                try
```

Comment:
I thought the api doesn't throw? Or is it MKNet that deals with that?

Reply:
Most MKNet API implementation is composed of calls to mk, where exceptions are handled and codes are returned. We can remove most of the try/finally blocks, but it needs to be consistent across all tests